randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1598,48 +1570,30 @@ Extra - map_name: rm_a5b15
 > Dock to Facility Storage Tower East; Heals? False
   * Layers: default
   * Open Passage to Facility Storage Tower East/Dock to Spiky Space Jump Trial Entrance
-  > Dock to Spiky Space Jump Trial
-      All of the following:
-          Gravity Suit and Speed Booster and Shinesparking Tricks (Beginner)
-          Any of the following:
-              # Climb Requirements
-              Hi-Jump Boots or Power Grip or Space Jump or Walljump (Beginner)
-  > Bottom
-      Any of the following:
-          Power Grip or Movement (Beginner) or Walljump (Beginner)
-          Gravity Suit and Power Grip Wall
-          All of the following:
-              # IBJ
-              Infinite Bomb Jumping (Intermediate) and Can Use Bombs
-          All of the following:
-              # CBJ
-              Charged Bomb Jump (Beginner) and Can Use Charged Bomb Jump
+  > Center
+      Movement (Beginner) or Underwater Power Grip Wall
 
 > Dock to Spiky Space Jump Trial; Heals? False
   * Layers: default
   * Open Passage to Spiky Space Jump Trial/Dock to Spiky Space Jump Trial Entrance
-  > Bottom
-      Gravity Suit and Speed Booster
+  > Center
+      All of the following:
+          Gravity Suit and Speed Booster
+          Movement (Beginner) or Shinesparking Tricks (Beginner) or Power Grip Wall
 
 > Pickup (Missile Tank); Heals? False
   * Layers: default
   * Pickup 304; Category? Minor
   * Extra - object_name: oItemM_304
-  > Bottom
-      Trivial
+  > Center
+      Underwater Power Grip Wall
 
-> Bottom; Heals? False
+> Center; Heals? False
   * Layers: default
   > Dock to Facility Storage Tower East
-      Any of the following:
-          Power Grip or Walljump (Beginner)
-          Gravity Suit and Power Grip Wall
-          All of the following:
-              # IBJ
-              Infinite Bomb Jumping (Intermediate) and Can Use Bombs
-          All of the following:
-              # CBJ
-              Charged Bomb Jump (Beginner) and Can Use Charged Bomb Jump
+      Trivial
+  > Dock to Spiky Space Jump Trial
+      Gravity Suit and Speed Booster and Shinesparking Tricks (Beginner)
```

Comment:
I'm wondering whether this should be bumped to intermediate, since it's not as simple as "crouch, and then spark". You have to shoot blocks, then jump a bit to get into position, and *then* spark.